survey-runner-application/survey_runner.tf

```diff
@@ -232,6 +232,61 @@ resource "aws_elastic_beanstalk_environment" "survey_runner_prime" {
     name      = "EQ_SUBMITTED_RESPONSES_TABLE_NAME"
     value     = "${var.submitted_responses_table_name}"
   }
+  setting {
+    namespace = "aws:elasticbeanstalk:application:environment"
+    name      = "EQ_SUBMITTED_RESPONSES_DYNAMO_READ"
+    value     = "True"
```

Comment:
This is defaulted to `True` so we shouldn't set to what is already default.
Also we shouldn't need this one at all